browser-tests/functional-tests/example.spec.js

```diff
@@ -0,0 +1,63 @@
+const { test, expect } = require("@playwright/test");
+
+test("Example page navigation", async ({ page }) => {
```

Comment:
can these be 'real' rather than example tests? the intention is just to have one set of page rendering scenarios like these, rather than building out scenarios for all the pages/journeys